backend/app/routes/recommend.py

```diff
@@ -41,40 +39,32 @@ async def recommend_post(
 ):
     """Do a search against the specified collection, pass results to RS"""
```

Comment:
Wouldn't be better to call the function `get_collection_recommendations` instead of adding the comment?